src/components/site/SummaryList.md

```diff
@@ -0,0 +1,133 @@
+```jsx
+const publicWorksSummaryItems = [
+  {
+    id: 1,
+    icon: "fas fa-map-marker-alt",
+    heading: "Main Office",
+    content:
+      "<p> 111 West Chesapeake Avenue<br /> Suite 307<br /> Towson, Maryland 21204</p>"
+  },
+
+  {
+    id: 2,
+    icon: "far fa-clock",
+    heading: "Hours:",
+    content: "<p>Monday through Friday<br />8 a.m. to 4:30 p.m.</p>"
+  },
+  {
+    id: 3,
+    icon: "far fa-envelope",
+    heading: "Email:",
+    content: "<p><a href=`#`>publicworks@​baltimorecountymd.gov</a></p>"
+  },
+  {
+    id: 4,
+    icon: "fas fa-mobile-alt",
+    heading: "Phone:",
+    content: "<p>410-887-6789</p><p>Emergency Dispatch:</p><p>410-887-6789</p>"
+  },
+  {
+    id: 5,
+    icon: "fas fa-exclamation-circle",
+    heading: "BALTCOGO311",
+    content: " <p><a href=`#`>Report a problem</a> online.</p>"
+  },
+  {
+    id: 6,
+    icon: "fas fa-user",
+    heading: "Director:",
+    content: "<p>Steve Walsh, P.E.</p>"
+  }
+];
+
+<SummaryList items={publicWorksSummaryItems} />;
+```
+
+Html Snippet:
+
+```html
+<div>
+  <div class="dg_summary-list">
+    <div class="dg_summary-list-items">
+      <aside class="icon icon-1">
+        <i class="fas fa-map-marker-alt" aria-hidden="true"></i>
+      </aside>
+      <div class="heading"><h3>Main Office</h3></div>
+    </div>
+    <div class="dg_summary-list-items">
+      <div class="push">
+        <p>
+          111 West Chesapeake Avenue<br />
+          Suite 307<br />
+          Towson, Maryland 21204
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="dg_summary-list">
+    <div class="dg_summary-list-items">
+      <aside class="icon icon-1">
+        <i class="far fa-clock" aria-hidden="true"></i>
+      </aside>
+      <div class="heading"><h3>Hours:</h3></div>
+    </div>
+    <div class="dg_summary-list-items">
+      <div class="push">
+        <p>Monday through Friday<br />8 a.m. to 4:30 p.m.</p>
+      </div>
+    </div>
+  </div>
+  <div class="dg_summary-list">
+    <div class="dg_summary-list-items">
+      <aside class="icon icon-1">
+        <i class="far fa-envelope" aria-hidden="true"></i>
+      </aside>
+      <div class="heading"><h3>Email:</h3></div>
+    </div>
+    <div class="dg_summary-list-items">
+      <div class="push">
+        <p><a href="`#`">publicworks@​baltimorecountymd.gov</a></p>
+      </div>
+    </div>
+  </div>
+  <div class="dg_summary-list">
+    <div class="dg_summary-list-items">
+      <aside class="icon icon-1">
+        <i class="fas fa-mobile-alt" aria-hidden="true"></i>
+      </aside>
+      <div class="heading"><h3>Phone:</h3></div>
+    </div>
+    <div class="dg_summary-list-items">
+      <div class="push">
+        <p>410-887-6789</p>
```

Comment:
More of a question. Would it be worth allowing the phone number(s) to be wrapped in a link with tel:? IE. <a href="tel:410-887-6789">410-887-6789</a>

Reply:
done